make/modules/java.desktop/lib/Awt2dLibraries.gmk

```diff
@@ -232,11 +232,17 @@ ifeq ($(call isTargetOs, windows macosx), false)
         WARNINGS_AS_ERRORS_xlc := false, \
         DISABLED_WARNINGS_gcc := int-to-pointer-cast, \
         DISABLED_WARNINGS_gcc_awt_Taskbar.c := parentheses, \
+        DISABLED_WARNINGS_clang_awt_Taskbar.c := parentheses, \
```

Comment:
please group the disabled warnings by compiler (gcc, then clang, then clang_aix)

Reply:
Done